packages/transit-vehicle-overlay/src/VehicleMarker/tracker.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { vehicleType } from "../types";
+import { Button } from "./styled";
+
+/**
+ * presentational component for tracking button on marker popup
+ */
+function VehicleTracker(props) {
+  const { vehicle } = props;
+  const { tracked } = props;
+  const { setTracked } = props;
+
+  function handleClick() {
```

Comment:
Seeing this function and the next makes me think the VehicleTracker should be converted into a Component. Otherwise, these functions are defined all over again on every render.